src/controller/python/matter/clusters/attribute.cpp

```diff
@@ -360,7 +360,7 @@ PyChipError pychip_WriteClient_WriteAttributes(void * appContext, DeviceProxy *
     std::unique_ptr<WriteClientCallback> callback = std::make_unique<WriteClientCallback>(appContext);
     std::unique_ptr<WriteClient> client           = std::make_unique<WriteClient>(
         app::InteractionModelEngine::GetInstance()->GetExchangeManager(), callback->GetChunkedCallback(),
-        timedWriteTimeoutMs != 0 ? Optional<uint16_t>(timedWriteTimeoutMs) : Optional<uint16_t>::Missing());
+        timedWriteTimeoutMs != 0 ? Optional<uint16_t>(timedWriteTimeoutMs) : Optional<uint16_t>::Missing(), suppressResponse);
```

Comment:
So I think the main concern I have with this change is that the client-side implementation of SuppressResponse is broken, at least in situations that involve chunking.

Therefore, any use of this code with `suppressResponse` set to true and chunking involved will produce incorrect packets on the wire.  This is a hidden footgun that we should ideally not create.

As in, we should fix the client-side SuppressResponse behavior either in this PR or in a separate PR before it.